opam-ci-check/lib/lint.ml

```diff
@@ -265,13 +265,20 @@ module Checks = struct
         if is_build then (pkg, DuneIsBuild) :: errors else errors
     | None -> []
 
+
   let check_maintainer_email ~pkg opam =
```

Comment:
nit: Maybe also rename the check function?

Reply:
Good catch. Thanks.